src/display/api.js

```diff
@@ -1097,7 +1097,7 @@ var PDFWorker = (function PDFWorkerClosure() {
     if (typeof PDFJSDev !== 'undefined' &&
         PDFJSDev.test('PRODUCTION && !(MOZCENTRAL || FIREFOX)') &&
         pdfjsFilePath) {
-      return pdfjsFilePath.replace(/\.js$/i, '.worker.js');
+      return pdfjsFilePath.replace(/(\.(?:min\.)?js)$/i, '.worker$');
```

Comment:
Please note that this there's a `1` missing at the end of the newSubString parameter, since as mentioned in https://github.com/mozilla/pdf.js/pull/8256#pullrequestreview-31664316 it should be `.replace(/(\.(?:min\.)?js)$/i, '.worker$1')` in order to actually work.